archive/v1.17/docs/ops/performance-and-scalability/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html lang=en><head><title>Redirecting&mldr;</title><link rel=canonical href=/v1.17/docs/ops/deployment/performance-and-scalability/><meta name=robots content="noindex"><meta charset=utf-8><meta http-equiv=refresh content="0; url=/v1.17/docs/ops/deployment/performance-and-scalability/"></head><body><h1>Redirecting&mldr;</h1><a href=/v1.17/docs/ops/deployment/performance-and-scalability/>Click here if you are not redirected.</a></body></html>
```

Comment:
I'm not sure why this file is included.

Reply:
wasn't sure if we wanted to do 1.17 data too

Reply:
If we wanted 1.17 data, we should do it in a release-1.17 PR and then have it re-archived to 1.18 in another PR. At this point, I feel we can ignore that.